reader_controller.py

```
@@ -0,0 +1,26 @@
+import zipfile
```

Ruff (0.11.9)

`zipfile` imported but unused

Remove unused import: `zipfile`

(F401)